Teleport AI (Maybe Working)

inburst21 opened the pull request:
This new Teleport AI will Learn as it goes. It still needs some more
work but I'm almost ready for release. It generates a new file called
TeleAI.json in the config file. It can load the file but im having
trouble saving the new delays. As soon as that is done it is ready for
release.

inburst21 committed:
Teleport AI (Need Help)

This new Teleport AI will Learn as it goes. It still needs some more
work but I'm almost ready for release. It generates a new file called
TeleAI.json in the config file. It can load the file but im having
trouble saving the new delays. As soon as that is done it is ready for
release.

inburst21 committed:
Removed Unused code

It has some Cleaning to do but im not sure how saving or loading works.
So not sure what I have to keep

inburst21 committed:
Code Cleanup - Maybe Working

Remove unneeded code. May be working at this time but uncertain due to
not being able to test.

inburst21 committed:
Random wait time

Added Random wait time just incase their tracked

PocketMobsters commented:
When I get home from work I'll take a closer look at this but first off - TeleportAI.cs doesn't really comply to our standards, check out the event logging and such to see how we prefer any console output to work, we don't want to put any actual Logger.WriteLine or console.WriteLine code outside of those handlers since it'll mess with GUI development.

inburst21 commented:
I was actually adding it to the event class. I didn't even see that comment before I finished all my work today. It should be up to standards now?  I have fixed it up a lot but I know there is a way to compress all of that into a small method like maybe 7 lines a piece for both. I am just not sure how to do it.

inburst21 committed:
Random Teleports and Added Translation

I removed an unneeded class and made it so the teleports no longer land
directly on the stop. Also added it to translations. Ready for release
if testing works when its back up

inburst21 committed:
Revert "Random Teleports and Added Translation"

This reverts commit a9c10c4759cc55a6465c9ca6ddb65db470a86032.

inburst21 committed:
Revert "Revert "Random Teleports and Added Translation""

This reverts commit ec00e2d623366b8078bec5784d9cf79b5f88389a.

inburst21 committed:
Revert "Revert "Revert "Random Teleports and Added Translation"""

This reverts commit 1de219dff610587d8da4266131b54c093a202dfb.

inburst21 committed:
Revert "Revert "Revert "Revert "Random Teleports and Added Translation""""

This reverts commit 7b56961f2961b24a2e2b0acc676b108da2e1c3f9.

inburst21 committed:
Merge pull request #1 from inburst21/dev

Random Teleport and Translations

inburst21 committed:
Revert "Random Teleport and Translations"

inburst21 committed:
Merge pull request #2 from inburst21/revert-1-dev

Revert "Random Teleport and Translations"
